R/confirmatoryfactoranalysis.R

```diff
@@ -193,16 +190,12 @@ confirmatoryFactorAnalysisInternal <- function(jaspResults, dataset, options, ..
   geq <- .CFAInvariance(options)
   if (options$group == "") grp <- NULL else grp <- options$group
 
-  # define estimator from options
-  estimator = switch(options[["estimator"]],
-                     "default"                         = "default",
-                     "maximumLikelihood"               = "ML",
-                     "generalizedLeastSquares"         = "GLS",
-                     "weightedLeastSquares"            = "WLS",
-                     "unweightedLeastSquares"          = "ULS",
-                     "diagonallyWeightedLeastSquares"  = "DWLS"
-  )
-
+  if (anyNA(dataset)) {
+    naAction <- ifelse(options$naAction == "twoStageRobust", "robust.two.stage",
+                       ifelse(options$naAction == "twoStage", "two.stage", options$naAction))
```

Comment:
I think this would be prefered: 
```suggestion
    naAction <- switch(
      options$naAction, 
      "twoStageRobust" = "robust.two.stage",
      "twoStage"       = "two.stage",
      options$naAction)
```